### ISSUE TITLE
Stuck in `Your selection:`

### ISSUE THREAD

**Issue by hissssst** [hexpm/hex#969]
```
You are a member of one or multiple organizations. Would you like to publish the package with yourself as owner or an organization as owner? If you publish with an organization as owner your package will be public but managed by the selected organization.

  [1] Yourself
  [2] clickhouse_elixir
  [3] secret_vault

Your selection: 3
Your selection: 3
Your selection: 3
```

```bash
$ mix hex.info
Hex:    2.0.0
Elixir: 1.13.4
OTP:    24.3.4.5

Built with: Elixir 1.13.4 and OTP 22.3
```

**Comment by hissssst** [comment]
@ericmj I can see this was fixed in #967, but could you please release it any time soon? I need to publish two packages and I can't do this with this bug

**Comment by hissssst** [comment]
Worked around with `mix archive.install github hexpm/hex branch main`
